demo/ibis-demo.es.qmd

```diff
@@ -0,0 +1,258 @@
+---
+title: Demo de Ibis
+---
+
+Primero, importar Ibis:
+
+```{python}
+import ibis
+from ibis import *
+```
+
+## Conectar a DuckDB
+
+```{python}
+con = ibis.duckdb.connect()
+```
+
+## Cargar unos datos adentro de las tablas
+
+
+```{python}
+!curl -OLsS 'https://storage.googleapis.com/ibis-tutorial-data/imdb/2024-03-22/imdb_title_ratings.parquet'
+!curl -OLsS 'https://storage.googleapis.com/ibis-tutorial-data/imdb/2024-03-22/imdb_title_basics.parquet'
+```
+
+```{python}
+ratings = con.read_parquet("imdb_title_ratings.parquet", table_name="imdb_title_ratings")
+basics = con.read_parquet("imdb_title_basics.parquet", table_name="imdb_title_basics")
+```
+
+Podemos presentar los nombres de las tablas
+
+```{python}
+con.list_tables()
+```
+
+Y las renderizamos la expresión abstracta:
```

Comment:
```suggestion
Y las renderizamos con la expresión abstracta:
```
"And we render them with the abstract expression"
Or if we want to say "and we render the abstract expression" it'll be 

> Y renderizamos la expresión abstracta